app/src/pages/inside/logsPage/logItemInfo/defectDetails/defectDetails.jsx

```diff
@@ -53,6 +53,7 @@ import {
   IgnoredInAALabel,
   PALabel,
 } from 'pages/inside/stepPage/stepGrid/defectType/defectType';
+import { DEFECT_EDITOR_MODAL } from 'pages/inside/stepPage/modals/editDefectModals/constants';
```

Comment:
Could you please rename this modal to `MAKE_DECISION_MODAL` according to its purpose, it is not just a defect editor.